src/main/java/org/example/intro/dto/cart/CartItemDto.java

```diff
@@ -0,0 +1,13 @@
+package org.example.intro.dto.cart;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
```

Comment:
not a mistake - it's ok to just use @Data on Dtos, nothing dangerous here (unlike entities)

Reply:
++, updated. thank you.